bokeh/models/tools.py

```diff
@@ -704,8 +705,8 @@ class HoverTool(Inspection):
     have a matching value for their ``name`` attribute will be used.
     """)
 
-    renderers = List(Instance(Renderer), help="""
-    An explicit list of renderers to hit test again. If unset,
+    renderers = Either(Auto, List(Instance(Renderer)), help="""
```

Comment:
The default value of "auto" isn't set here, or the help text is wrong if it's meant to default to no renderers.